AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/AWSMutationDatabaseAdapterTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright 2018-2019 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import XCTest
+
+@testable import Amplify
+@testable import AmplifyTestCommon
+@testable import AWSDataStoreCategoryPlugin
+
+class AWSMutationDatabaseAdapterTests: XCTestCase {
+    var databaseAdapter: AWSMutationDatabaseAdapter!
+
+    let model1 = Post(title: "model1", content: "content1")
+
+    override func setUp() {
+        do {
+            let mockStorageAdapter = MockSQLiteStorageEngineAdapter()
+            databaseAdapter = try AWSMutationDatabaseAdapter(storageAdapter: mockStorageAdapter)
+        } catch {
+            XCTFail("Failed to setup sut")
```

Comment:
What's a 'sut'? :)

Reply:
sut is system under test -- i can spell this out, or we can adopt sut

Reply:
updated wording to `system under test`

Reply:
Got it. I don't object to it, I just don't use the term regularly, so I didn't recognize it as an acronym without the all-caps spelling :)